src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -321,8 +324,25 @@ object SonarLintIntelliJClient : SonarLintClient {
     override fun assistCreatingConnection(params: AssistCreatingConnectionParams): CompletableFuture<AssistCreatingConnectionResponse> {
         return CompletableFuture.supplyAsync {
             val serverUrl = params.serverUrl
-            val message = "No connections configured to '$serverUrl'."
-            if (!showConfirmModal(OPENING_FINDING_TITLE, message, "Create connection", null)) {
+            val warningTitle = "Do you trust this SonarQube server?"
```

Comment:
We could move this constant to companion object as WARNING_TITLE

Reply:
This is only used in this method so I'd lean towards keeping the variable in the method where it is used, WDYT?